### DISCUSSION TITLE
Task 1

### DISCUSSION THREAD

**Discussion by arunsundar022** [arunsundar022/Corona-tweet-sentiment-analysis#3] [General]
Divish you can start reading the dataset and let us know when you are done.

**Comment by Divishkumar28** [comment]
Guys I have finished my part of work, @Sainath26  can continue the next part of work (i.e. Visualization) 
@Sainath26 when you do your part, clone the code form branch Divi so that you could continue the project form where I had finished reading the dataset and also when you are done your part, upload your code in same branch (Divi) or create a new branch and upload your code
